node_modules/oae-core/editprofile/css/editprofile.css

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#editprofile-plea {
+    font-size: 14px;
+    font-style: normal;
+    margin-bottom: 15px;
+}
+
+#editprofile-email-verification .alert {
```

Comment:
There's no need for this widget to use flexbox. We should be able to achieve everything using standard bootstrap and floats